coriolis/tests/test_cache.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+import unittest
+from unittest import mock
+from coriolis import exception
+from coriolis import cache
+
+
+class TestGetCacheDecorator(unittest.TestCase):
```

Comment:
Base class of `TestGetCacheDecorator`, and all future test classes, should be `tests.test_base.CoriolisBaseTestCase`. Therefore, please:

```python
from tests import test_base
#...
class TestGetCacheDecorator(test_base.CoriolisBaseTestCase):
```

Reply:
Good to know! Thanks a lot!